OWML.ModHelper.Menus/ModInputCombinationElement.cs

```diff
@@ -0,0 +1,179 @@
+
+namespace OWML.ModHelper.Menus
+{
+    public class ModInputCombinationElement : ModToggleInput, IModInputCombinationElement
+    {
+        private const float ScaleDown = 0.75f;
+
+        public ILayoutManager Layout { get; private set; }
+
+        override public string Title
+        {
+            get => _combination;
+            set
+            {
+                _combination = value;
+                UpdateContents();
+            }
+        }
+
+        private string _combination;
+        private readonly GameObject _layoutObject;
+        private readonly IModInputHandler _inputHandler;
+
+        private static IModInputCombinationElementMenu _popupMenu;
+
+        private void SetupButtons()
+        {
+            var commandObject = new GameObject();
+            var commandComponent = commandObject.AddComponent<ModCommandListener>();
+            commandComponent.Initialize(InputLibrary.interact);
+            commandComponent.OnNewlyReleased += OnEditButton;
+            YesButton.Title = "Edit";
+            YesButton.OnClick += OnEditClick;
+
+            var deleteCommand = new SingleAxisCommand();
+            var deleteBindingGmpd = new InputBinding(JoystickButton.FaceUp);
+            var deleteBindingKbrd = new InputBinding(KeyCode.Delete);
+            deleteCommand.SetInputs(deleteBindingGmpd, deleteBindingKbrd);
+            commandObject = new GameObject();
+            var updater = commandObject.AddComponent<ModCommandUpdater>();
+            updater.Initialize(deleteCommand);
+            commandComponent = commandObject.AddComponent<ModCommandListener>();
+            commandComponent.Initialize(deleteCommand);
+            commandComponent.OnNewlyReleased += OnDeleteButton;
+            NoButton.Title = "Delete";
+            NoButton.OnClick += OnDeleteClick;
+        }
+
+        public ModInputCombinationElement(TwoButtonToggleElement toggle, IModMenu menu,
+            IModInputCombinationElementMenu popupMenu, IModInputHandler inputHandler, string combination = "") :
+            base(toggle, menu)
+        {
+            _inputHandler = inputHandler;
+            _combination = combination;
+            _layoutObject = toggle.GetComponentInChildren<HorizontalLayoutGroup>().transform.Find("LabelBlock").
+                GetComponentInChildren<HorizontalLayoutGroup>().gameObject;
+            var layoutGroup = _layoutObject.GetComponent<HorizontalLayoutGroup>();
+            var scale = toggle.transform.localScale;
+
+            SetupButtons();
+
+            Initialize(menu);
+            layoutGroup.childControlWidth = false;
+            layoutGroup.childControlHeight = false;
+            layoutGroup.childForceExpandHeight = false;
+            layoutGroup.childForceExpandWidth = false;
+            layoutGroup.spacing = 0f;
+            var constantGraphics = _layoutObject.GetComponentsInChildren<Graphic>(true);
+            _layoutObject.transform.GetChild(1).gameObject.SetActive(false);
```

Comment:
find by name / component here too instead of `GetChild()`

Reply:
Same for any other `GetChild()` in this PR, better to avoid it completely if possible.

Reply:
Got rid of them